commons/src/main/java/org/frankframework/util/UUIDUtil.java

```diff
@@ -100,7 +101,7 @@ public static String createNumericUUID() {
 		String ia4 = df.format(unsignedByteToInt(ipAddress[3]));
 		String ia = ia1 + ia2 + ia3 + ia4;
 
-		long hashL = Math.round(Math.random() * 1000000);
+		long hashL = Math.round(new SecureRandom().nextDouble() * 1_000_000d);
```

Comment:
Dit is dus typisch een geval waar je 1 centraal static instance zou willen hebben, en niet iedere keer een nieuwe maken.